src/Compilers/CSharp/Portable/Binder/Binder_InterpolatedString.cs

```diff
@@ -153,13 +154,23 @@ private BoundExpression BindInterpolatedString(InterpolatedStringExpressionSynta
                         case SyntaxKind.InterpolatedStringText:
                             {
                                 var text = ((InterpolatedStringTextSyntax)content).TextToken.ValueText;
-                                builder.Add(new BoundLiteral(content, ConstantValue.Create(text, SpecialType.System_String), stringType));
+                                // Raw string literals have no escapes.  So there is no need to manipulate their value texts.
+                                // We have to unescape normal interpolated strings as the parser stores their text without
+                                // interpretting {{ and }} sequences (as '{' and '}') respectively.  Changing that at the syntax
+                                // level might potentially be a breaking change, so we do the conversion here when creating the
+                                // bound nodes.
+                                if (!isRawInterpolatedString)
+                                {
+                                    text = ConstantValueUtils.UnescapeInterpolatedStringLiteral(text);
+                                }
```

Comment:
note: i'm also amenable to actually changing the .ValueText for a normal interpolated string.  I was actually *super* surprised to discover that `{{` in a text chunk is treated as `{{` in the `.ValueText`!   Surprised to the point that it feels like a bug that we ended up working around in later portions of the code, as nothing else seems to do this.